apps/survey/src/components/prompts/layouts/base-layout.vue

```diff
@@ -124,12 +118,16 @@ import type { PropType } from 'vue';
 import { defineComponent } from 'vue';
 
 import type { EncodedFood, MissingFood } from '@intake24/common/types';
+import MealListMobile from '@intake24/survey/components/layouts/meal-list/list-mobile.vue';
+import ReviewList from '@intake24/survey/components/layouts/meal-list/review-list.vue';
```

Comment:
I think unused import, can be removed

You can also use barrel import
```
import { MealListMobile } from '@intake24/survey/components/layouts/meal-list';
```

Reply:
I will remove all the unused imports and use barrel import.